source/localizable/csvspec/index.en.html.md.erb

```diff
@@ -0,0 +1,468 @@
+---
+title: csvspec
```

Comment:
Although this is more readable, restructuring of the urls are outside the scope of this. We will break bookmarks and links to feedspec from other documents like `ecommerce.html.md.erb`